src/coffee/run.coffee

```diff
@@ -91,7 +91,9 @@ module.exports = class
       .option '-s, --clientSecret <secret>', 'your OAuth client secret for the SPHERE.IO API'
       .option '--accessToken <token>', 'an OAuth access token for the SPHERE.IO API, used instead of clientId and clientSecret'
       .option '--sphereHost <host>', 'SPHERE.IO API host to connect to'
+      .option '--sphereProtocol <protocol>', 'SPHERE.IO API protocol to connect to'
```

Comment:
then we do the same for `sphereHost`, `sphereAuthHost` and `sphereAuthHost`?
I am not for it.